# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-disable-nodejs-version-check.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-136.0.7103.33/third_party/node/node.gni.vanilla	2025-04-19 11:29:52.884513409 +0200
+++ chromium-136.0.7103.33/third_party/node/node.gni	2025-04-19 11:31:36.201682707 +0200
@@ -35,16 +35,5 @@
         inputs += [ "//third_party/node/mac/node-darwin-x64/bin/node" ]
       }
     }
-
-    # Automatically add a dependency to ":check_version" to ensure NodeJS is
-    # always running the expected version, except when the ':check_version'
-    # target itself is running in which case it shouldn't depend on itself.
-    if (get_label_info(":" + target_name, "label_no_toolchain") !=
-        "//third_party/node:check_version") {
-      if (!defined(deps)) {
-        deps = []
-      }
-      deps += [ "//third_party/node:check_version" ]
-    }
   }
 }
--- chromium-138.0.7204.49/third_party/protobuf/proto_library.gni.vanilla	2025-06-27 07:44:30.475491176 +0200
+++ chromium-138.0.7204.49/third_party/protobuf/proto_library.gni	2025-06-27 07:44:49.689023768 +0200
@@ -562,7 +562,7 @@
                   _protoc_gen_ts_path,
                   "//tools/protoc_wrapper/protoc-gen-ts_proto.py",
                 ] + _protoc_gen_ts_runtime_deps
-      deps += [ "//third_party/node:check_version" ]
+      #deps += [ "//third_party/node:check_version" ]
     }
 
     if (_generate_with_plugin) {
